web/react/components/edit_post_modal.jsx

```diff
@@ -59,6 +61,18 @@ export default class EditPostModal extends React.Component {
     handleUserInput(e) {
         this.setState({editText: e.target.value});
     }
+    handleEditPostEvent(options) {
+        this.setState({
+            editText: options.message || '',
+            title: options.title || '',
+            post_id: options.postId || '',
```

Comment:
Yeah the post_id isn't being set currently, but that explains why I'm getting a server error when testing this